testdata/serviceconfig.yml

```diff
@@ -7,6 +7,8 @@ rules:
     params:
       quotas:
       - descriptor_name: RequestCount
```

Comment:
You know, in the spirit of the other changes we've been making to the config API, moving from messages with an embedded name field to a `map<name, other fields>`, should we do the same with our aspect params too?

```yaml
- selector: true
  aspects:
  -kind: quotas
    params:
      RequestCount:
        max_amount: 5
        expiration: 1s
      FooBar:
        max_amount: 1000000
        expiration: 1s
```

Though maybe this goes too far and makes it non-obvious that this is a reference to the "RequestCount" and "FooBar" quota descriptors.